cx/slack/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "slack-integration",
+  "version": "0.0.1",
+  "description": "A server that integrates Dialogflow CX with Slack.",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "test": "mocha --recursive --exit"
+  },
+  "dependencies": {
+    "@google-cloud/dialogflow-cx": "2.7.0",
+    "@slack/events-api": "^2.3.4",
+    "@slack/web-api": "^5.15.0",
+    "axios": "^0.21.1",
```

Comment:
Does this service use both `axios` and `request` package?

Reply:
It does not use neither axios or request, so I have removed them from dependencies.